CONTRIBUTING.md

```diff
@@ -3,4 +3,4 @@ Before submitting a pull request, please verify that:
 * There are tests added to make sure the bug does not return/feature does not break in the future.
 * Any strings are extracted for [translation](http://projects.theforeman.org/projects/foreman/wiki/Translating).
 
-For more information regarding contributions, plese read the [Foreman Handbook](http://theforeman.org/handbook.html).
+For more information regarding contributions, plese read the [Foreman Handbook](http://theforeman.org/handbook.html) and [Foreman Code-related contributions](http://theforeman.org/contribute.html#Code-relatedcontributions).
```

Comment:
Hi, thanks for the input! care to make that a link to the main contribute page (https://theforeman.org/contribute.html) rather then that specific subsection?